cobigen-templates/templates-devon4j/src/main/java/com/devonfw/cobigen/templates/devon4j/utils/JavaUtil.java

```diff
@@ -215,31 +207,6 @@ public String castJavaPrimitives(Class<?> pojoClass, String fieldName)
     }
   }
 
-  /**
-   * @param pojoClass {@link Class} the class object of the pojo
-   * @param fieldName {@link String} the name of the field
-   * @return true if the field is an instance of java.utils.Collections
-   * @throws NoSuchFieldException indicating something awefully wrong in the used model
-   * @throws SecurityException if the field cannot be accessed.
-   */
-  public boolean isCollection(Class<?> pojoClass, String fieldName) throws NoSuchFieldException, SecurityException {
-
-    if (pojoClass == null) {
-      return false;
-    }
-
-    Field field = pojoClass.getDeclaredField(fieldName);
-    if (field == null) {
-      field = pojoClass.getField(fieldName);
-    }
-    if (field == null) {
-      return false;
-    } else {
-      return Collection.class.isAssignableFrom(field.getType());
-    }
-
-  }
-
   /**
    * Returns the Ext Type to a given java type
```

Comment:
params and return are missing at the javadocs

Reply:
These methods are not adressed in this pull-request

Reply:
Not code pertaining to this PR.